core/src/test/java/org/arquillian/smart/testing/impl/TestStrategyApplierUsingPropertyTest.java

```diff
@@ -135,4 +140,9 @@ private TestExecutionPlannerLoader prepareLoader(final Set<Class<?>> testsToRun,
     private Set<Class<?>> createTestsToRun(Class... testClasses) {
         return new LinkedHashSet<>(Arrays.<Class<?>>asList(testClasses));
     }
+
+    private String getBaseDir(Class<?> aClass) {
```

Comment:
do we really need this?

Reply:
yes we need this, otherwise we have to hard code this value to "core" to work it from `IDEA` & terminal.